src/main/java/core/basesyntax/service/ReportGenerator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface ReportGenerator {
```

Comment:
Name doesn't match the logic:
```suggestion
public interface ReportWriter {
```